### DISCUSSION TITLE
pl

### DISCUSSION THREAD

**Discussion by giscus** [kjmjh/kjmjh.github.io#2] [Announcements]
# pl

https://kjmjh.github.io/pl.html

**Comment by kjmjh** [comment]
ohhhhhhhhhhhhhhhhhhhhhhhh

**Comment by lingbopro** [comment]
哦？用上giscus了？

**Comment by kjmjh** [comment]
那可不

**Reply by lingbopro** [reply]
虽然但是，这个讨论不改个名字吗

**Reply by kjmjh** [reply]
懒……(抠鼻屎)……你说啥？……

**Reply by JulyGlow** [reply]
这讨论真diao.Excellent!

**Reply by lingbopro** [reply]
有一种缺失本地翻译键的美（）

**Reply by kjmjh** [reply]
哈？

**Reply by kjmjh** [reply]
![无标题-2024-09-22-2010.png](https://github.com/user-attachments/assets/cdf787b8-4410-4939-b7a2-c824c9db7820)



<不要回复!>

**Reply by kjmjh** [reply]
bro我整天对着自己自言自语

**Comment by zsr-lukezhang** [comment]
这玩意改改一下了
你 B 站叫啥？

**Reply by kjmjh** [reply]
关注你了

**Reply by lingbopro** [reply]
@kjmjh 是 https://space.bilibili.com/3546773983332961

**Comment by bill74186** [comment]
# 陌生人的评论

> 一条陌生人发送的评论

一个与你**素不相识**的陌生人关注了你的网站

**Reply by kjmjh** [reply]
awa，你是怎么找到这里的?

**Reply by bill74186** [reply]
## 在一片茫茫之中

_(通过搜索.github.io)_

我看到你的网站，尤其是AI内容

我根据你的`ai.html`网页制作了一个**叫AI-Chat**的项目

网址是https://bill74186.github.io/AI-Chat

**能互相Star吗？**
